### DISCUSSION TITLE
How to enable federated timeline

### DISCUSSION THREAD

**Discussion by ghost** [mastodon/mastodon#17181] [Q&A]
I am a new mastodon administrator. I installed the software and I am the only user. How does one enable the "federated timeline"

My plan is to start posting as an admin and hopefully other mastodon instances can see my posts and join my instances.

When i click on "federated" i do not see any posts? How do i enable it so I can start seeing posts from other instances.

In the admin panel I dont see this option. The only option i see is "Add new relay" I tried adding one but it tells me i have to wait for the instances to accept. 

Does that mean I have to manually add every instance in the world and hopefully some of them add me? I thought this was supposed to be done automatically.

**Comment by trwnh** [comment]
it's automatic in the sense that the "federated timeline" shows all public posts *that your instance is aware of*. so for example, if you follow someone, their posts will show up in your home timeline, but their top-level and self-reply posts with "public" visibility will show up in everyone's federated timeline.

since you are the only user, the federated timeline is not going to be very useful for you; it is by design intended to show posts that other users may see in their home timelines.

however, as you point out, relays are a solution¹ to this. one point to mention, though: you do not add specific websites to the relay, but rather, you request to join a specific relay. an example list of relays can be found here: https://github.com/brodi1/activitypub-relays

¹ yes, this is not an ideal solution in its present form, sadly
